.github/workflows/ci.yml

```diff
@@ -120,17 +123,68 @@ jobs:
         run: |
           ddev composer install
           ddev pull platform -y
-          ddev drush deploy -y
+
+      - name: Run phpstan
+        run: ddev php ./vendor/bin/grumphp run --tasks=phpstan --no-interaction
+
+      - name: Rebuild cache
+        run: |
+          ddev drush cache:rebuild -y
+          ddev drush config:status
+
+      - name: Import Config and Run updates
+        run: |
+          ddev drush updatedb --no-cache-clear --yes
+          ddev drush config:import --yes
+          ddev drush cache:rebuild --yes
+          ddev drush deploy:hook --yes
```

Comment:
This is exactly the order documented in `drush deploy`. What's the value of breaking it down if we are going to use the same sequence?

---

From https://www.drush.org/12.x/deploycommand/

```
drush updatedb --no-cache-clear
drush cache:rebuild
drush config:import
drush cache:rebuild
drush deploy:hook
```

Reply:
We don't want drush cache:rebuild to run after updatedb. After updatedb, any config changes in update hooks are written to the database, but the cache is not cleared. As a result, config:import will only import any exported config that has been pushed and is different. If we clear the cache, the config cache is updated. If the config export is not pushed, it will show a difference in the config, and the old config will be imported.